src/components/Toolbar/index.js

```diff
@@ -126,6 +127,24 @@ export default class Toolbar extends Component {
     );
   }
 
+  renderInfo() {
+    return (
+      <div className={'toolbar--info'}>
+        <Popup icon="fa-info-circle" onBlur={this.handleBlur}>
+          <strong>How to Enter Answers:</strong>
+          <ul>
+            <li>
+              You can click a square once to enter an answer, and click that same square again to switch
+              between a down or across for that square
+            </li>
+            <li>You can also click the questions to go straight to answering them</li>
```

Comment:
`clues` instead of `questions`.

I think `go straight to answering them` is pretty clear, though ideally there'd be some consistency of how we reference the concepts of `cursor` and `selected orientation`. Unfortunately I don't have any clear favorites for how we explain these concepts, so fine to leave this is as is for now.